### PR TITLE
Merge master into 7.x 2019-10-29

### PR DESCRIPTION
**Pull request by urso**

### CODE REVIEW

**Comment by urso** [comment]
@dedemorton  please check [libbeat/docs/version.asciidoc](https://github.com/elastic/beats/pull/14298/files?file-filters%5B%5D=.asciidoc#diff-37781817cf614390dd472cec89d97747).

File: libbeat/docs/version.asciidoc [hunk 1]
```diff
@@ -1,7 +1,9 @@
-:stack-version: 8.0.0
-:doc-branch: master
+:stack-version: 7.6.0
+:doc-branch: 7.x
 :go-version: 1.12.12
 :release-state: unreleased
 :python: 2.7.9
 :docker: 1.12
 :docker-compose: 1.11
+:branch: 7.x
```
Comment (dedemorton) [inline libbeat/docs/version.asciidoc]: Delete this line - we're getting the setting for `branch` from the shared file in the docs repo now.

File: libbeat/docs/version.asciidoc [hunk 2]
```diff
@@ -1,7 +1,9 @@
...
+:ecs_version: 1.1
```
Comment (dedemorton) [inline libbeat/docs/version.asciidoc]: Delete this line - we're getting the setting for `ecs_version` from the shared file in the docs repo now.

**Comment by ph** [comment]
@steffen There seems to have quite a few modules failing in this backport for filebeat. Early check this might be related to a change in the format or a difference between master ES and 7.x ES.

**Pull request closed without merging**